css/home.css

```diff
@@ -291,12 +291,62 @@ Media Queries (overriding styles for different screen sizes)
 			url("../assets/images/pages/home/mobile/hero-home-mobile.jpg");
 	}
 
+	/* community partnerships */
+	.community-card .text-container {
+		width: 70%;
+	}
+	.community-card img {
+		object-fit: cover;
+		object-position: 70%;
+	}
+
+	.community-card a {
+		padding: var(--spacing-xs) 0;
+		text-align: center;
+		width: 100%;
+	}
+
+	/* global ecosystem */
+	.global-ecosystem .text-container {
+		width: 80%;
+		padding-right: 175px;
+	}
+
+	.global-ecosystem blockquote {
+		min-width: 340px;
+		top: 130px;
+		right: 125px;
+	}
+
```

Comment:
Block quote has been adjusted in positioning to look smoother especially at 900px --> 768px mq

![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/175de355-b4db-48ca-9389-2a17aaa3eb8e)

Reply:
Much better